env0/resource_environment.go

```diff
@@ -57,25 +57,21 @@ func resourceEnvironment() *schema.Resource {
 				Type:        schema.TypeBool,
 				Description: "should run terraform plan on pull requests creations",
 				Optional:    true,
-				Computed:    true,
 			},
 			"approve_plan_automatically": {
 				Type:        schema.TypeBool,
 				Description: "should deployments require manual approvals",
 				Optional:    true,
-				Computed:    true,
 			},
 			"deploy_on_push": {
 				Type:        schema.TypeBool,
 				Description: "should run terraform deploy on push events",
 				Optional:    true,
-				Computed:    true,
 			},
 			"auto_deploy_on_path_changes_only": {
 				Type:        schema.TypeBool,
 				Description: "redeploy only on path changes only",
 				Optional:    true,
-				Computed:    true,
```

Comment:
just so you know, if we set them (from the response value) theres a warning from terraform
(but we do that in other resources as well)

Reply:
Gonna have to live with that I guess